lib/assets/javascripts/views/plans/share.js

```diff
@@ -1,8 +1,13 @@
+import ariatiseForm from '../../utils/ariatiseForm';
+
+if ($) {
```

Comment:
Please, remove the if since webpack ensures that jquery is imported and consider calling passing to $ an anonymous arrow function, i.e. no need for ready.